packages/web/src/routes/sliders/sliders.tsx

```diff
@@ -64,6 +64,7 @@ export default function NutritionSlider(props: NutritionSliderProps) {
             max={max}
             name={name}
             onChange={handleChange}
+            onChangeCommitted={handleChange}
```

Comment:
For some reason this prop handles when the user clicks on the slider to change the value rather than dragging the thumb over.  Without it the value was sometimes not saved